ui/src/validators.ts

```diff
@@ -0,0 +1,9 @@
+import { Accessor } from 'solid-js'
+
+export function mustMatch(match: Accessor<string | undefined>) {
+  return (error: string): ((value: string | undefined) => string) => {
+    return (value: string | undefined) => {
+      return value !== match() ? error : ''
+    }
+  }
+}
```

Comment:
Where is this used?

Reply:
This is the password validator, the logic was used and reimplemented in multiple components so I added it seperatly